docs/docs/guides/template-backup-script.md

```diff
@@ -46,18 +46,18 @@ REMOTE_BACKUP_PATH="/path/to/remote/backup/directory"
 ### Local
 
 # Backup Immich database
-docker exec -t immich_postgres pg_dumpall -c -U postgres | /usr/bin/gzip > $UPLOAD_LOCATION/database-backup/immich-database.sql.gz
+docker exec -t immich_postgres pg_dumpall -c -U postgres > "$UPLOAD_LOCATION"/database-backup/immich-database.sql
```

Comment:
I think it is a good idea to leave gzip out, a lot of people run this on windows and such and it is simpler for that reason as well. It could still be a good idea to leave this original example un, perhaps commented out. What do you think?

Reply:
Added a comment with the original example, added `--rsyncable` as this does not significantly change backup size and makes it much easier to transport and chunk.